tst/testinstall/strings.tst

```diff
@@ -115,6 +121,12 @@ gap> PrintString(x);
 "'a'"
 gap> String(x);
 "'a'"
+gap> x:='\0x42';
+'B'
+gap> x:='\0xFF';
+'\377'
+gap> x:='\0xab';
+'\253'
 
```

Comment:
These test cases only cover positives, but not malformed inputs, which we should also test. E.g. `\0yAB`, `\0X12`, `\090`, `\0a0`, `\0x0`,  `\009`, `\00a`, `\00x`, `\0x1g`, ..